Text/Parsec/Language.hs

```diff
@@ -55,16 +55,17 @@ haskellStyle = emptyDef
 
 javaStyle  :: LanguageDef st
 javaStyle   = emptyDef
-		{ commentStart	 = "/*"
-		, commentEnd	 = "*/"
-		, commentLine	 = "//"
-		, nestedComments = True
-		, identStart	 = letter
-		, identLetter	 = alphaNum <|> oneOf "_'"
-		, reservedNames  = []
-		, reservedOpNames= []
+                { commentStart   = "/*"
+                , commentEnd     = "*/"
+                , commentLine    = "//"
+                , nestedComments = True
+                , identStart     = letter
+                , identLetter    = alphaNum <|> oneOf "_'"
+                , reservedNames  = []
+                , reservedOpNames= []
                 , caseSensitive  = False
-		}
+                }
+
```

Comment:
@albertnetymk, Well, I've added it because there is some sort of pattern in the source code organization in this file. There is header, then section of code, then two empty lines before next header. This was the only place where this pattern was broken (or maybe I'm seeing patterns that do not exist!). Anyway it's way too minor.